:art: Create PoC Vue-component in typescript (#2)
tlbootcamp/vuepress-plugin-contributors: README.md

## Usage

> You can use component as itself: just write `<Contributors/>` in .md file and vuala.

Reviewer: а вчом смысол
Что за кейс когда без инжектинга списка контрибуторов можно юзать?

Author: Просто когда важно в файле указать список контрибьюторов, но не хочется натягивать отдельную тему. Например: https://vuepress.tools/plugins/vuepress-plugin-contributors.html

> You can use component as itself: just write `<Contributors/>` in .md file and vuala.

Reviewer: ```suggestion
You can use the component by itself: add `<Contributors/>` to any `.md` file and voila.
```

But most valuable part of this plugin is ability to inject contributor list to each page.

> Unfortunately, vuepress allows customize Page component only via custom themes, so you need to do some magic.

Reviewer: ```suggestion
But the most valuable part of this plugin is the ability to inject a distinct contributor list into each page.
Unfortunately, VuePress allows to customize the `Page` component only via custom themes, so you need to do some magic as follows.
```

Create `theme` directory in your `.vuepress`, fill it with `index.js` and `layouts/Layout.vue` as in vuepress-example.

> Then add this theme as file dependency to a project (example), and setup your theme to config.js (example).

Reviewer: ```suggestion
Create `theme` directory in your `.vuepress`, `index.js` and `layouts/Layout.vue` there as in (vuepress-example)[vuepress-example].
Then add this theme as file dependency to a project (example), and setup your theme to config.js (example).
```

Reviewer: >(example)

Это типа тудушка?

Author: Да, поставлю туда прямые ссылки. Написал ридми чтобы просто не забыть за неделю всё что делал)

> If you want change size or place of Contributors component in Page component, please use `Layout.vue` and theme customizing instructions.

Reviewer: ```suggestion
If you to want change size or position of the `Contributors` component in the `Page` component, please use `Layout.vue` and (theme customization instruction)[https://vuepress.vuejs.org/theme/writing-a-theme.html].
```

Reviewer: Не дождался ответа в личке, поправь если ссылка не та
